villainary-react/app/forms/validators/textFieldValidators.ts

```diff
@@ -0,0 +1,10 @@
+export const trimWhitespaceValidator = (value: string) => {
+    if (!value || !value.trim()) {
+        return 'This field is required';
+    }
+    const trimmedValue = value.replace(/\s/g, '');
```

Comment:
would this disallow any spaces anywhere in the string?

Reply:
Yes this will remove all spaces as it has the global modifier of '/g'